src/odemis/acq/fastem.py

```diff
@@ -197,6 +210,14 @@ def _configure_overview_hw(scanner):
         logging.warning("Unexpected resolution %s on e-beam scanner, expected %s",
                         scanner.resolution.value, TILE_RES)
 
+    md = scanner.getMetadata()
+    if model.MD_SINGLE_BEAM_ROTATION in md:
+        scanner.rotation.value = md[model.MD_SINGLE_BEAM_ROTATION]
+    else:
+        scanner.rotation.value = SINGLE_BEAM_ROTATION_DEFAULT
+        logging.warning("Scanner doesn't have SINGLE_BEAM_ROTATION metadata, using %s rad.",
+                        scanner.rotation.value)
+
```

Comment:
For the case of single beam, I am pretty sure you should also set scanner metadata MD_ROTATION_COR to -rot so that the image is still shown aligned in Odemis GUI.

Reply:
done